packages/clay-card/src/Description.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import React from 'react';
+
+import classNames from 'classnames';
+
+import Context from './Context';
+
+import {CardDescriptionDisplayType} from './types';
```

Comment:
Do we need to keep it within `types` even?

Reply:
Nope.